.github/workflows/main.yml

```diff
@@ -16,14 +16,18 @@ jobs:
       matrix:
         features: ["unstable", "unstable force-inprocess", "unstable memfd"]
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
       - name: Install nightly toolchain
-        uses: actions-rs/toolchain@v1
+        uses: dtolnay/rust-toolchain@nightly
         with:
-          profile: minimal
-          toolchain: nightly
-          override: true
+          components: rustfmt, clippy
+
+      - name: rustfmt
+        run: cargo fmt --features "${{ matrix.features }}" 
```

Comment:
This should probably run in `check` mode in order to fail on the CI if the input isn't formatted?

Reply:
@mrobinson hello! thank you for the review, yes, i can.

Reply:
First PR: https://github.com/servo/ipc-channel/pull/324. After merge it I will create new PR with clippy.